configs/defs/rocky.textproto

```diff
@@ -0,0 +1,1179 @@
+      "        recursive: true"
+      "        filename_regex: \"systemd-journal-remote.service\""
+      "      }"
+      "    }"
+      "    files_to_check:{"
+      "      files_in_dir:{"
+      "        dir_path:\"/etc/systemd/system\""
+      "        recursive: true"
+      "        filename_regex: \"systemd-journal-remote.socket\""
+      "      }"
+      "    }"
+      "    existence:{should_exist: false}"
+      "  }"
+      "}}"
+  }
+}
+benchmark_configs: {
+  id: "journald-service-enabled"
+  compliance_note: {
+    version: { cpe_uri: "cpe:/o:rockylinux:rockylinux:8.5" version: "1.0.0" benchmark_document: "CIS Rocky Linux 8" }
+    title: "Ensure journald service is enabled"
+    description:
+      "Ensure that the systemd-journald serviceis enabled to allow capturing of logging events."
+    rationale:
+      "If the systemd-journald serviceis not enabled to start on boot, the system will not capture "
+      "logging events."
+    remediation:
+      "By default the systemd-journald service does nothavean [Install] sectionand thus "
+      "cannot be enabled / disabled.\nIt is meant to be referenced as Requires or wants by other "
+      "unit files. As such, if the status of systemd-journald isnot static, investigate why."
+    cis_benchmark: {
+      profile_level: 1
+      severity: LOW
+    }
+    scan_instructions:
+      "generic:{"
+      "  check_alternatives:{"
+      "    file_checks:{"
+      "      files_to_check:{files_in_dir:{"
+      "         dir_path:\"/etc/systemd/system/\""
+      "        recursive: true"
+      "        filename_regex: \"systemd-journal.service\""
+      "       }"
+      "      }"
+      "      existence:{should_exist:true}"
+      "    }"
+      "}}"
+  }
+}
+benchmark_configs: {
+  id: "journald-default-file-permissions-configured"
+  compliance_note: {
+    version: { cpe_uri: "cpe:/o:rockylinux:rockylinux:8.5" version: "1.0.0" benchmark_document: "CIS Rocky Linux 8" }
+    title: "Ensure journald default file permissions configured"
+    description:
+      "Journald will create logfiles that do not already existon the system. This setting controls "
+      "what permissions will be applied to these newly created files."
+    rationale:
+      "It isimportantto ensure that log files have the correct permissions to ensure that sensitive "
+      "data is archived and protected."
+    remediation:
+      "If the default configuration is not appropriate for the site specific requirements, copy "
+      "/usr/lib/tmpfiles.d/systemd.conf to /etc/tmpfiles.d/systemd.conf and modifyas "
+      "required. Requirements is either 0640 or site policy if that is less restrictive."
+    cis_benchmark: {
+      profile_level: 1
+      severity: LOW
+    }
+    scan_instructions:
+      "generic:{"
+      "  check_alternatives:{"
+      "    file_checks:{"
+      "      files_to_check:{single_file:{"
+      "        path:\"/usr/lib/tmpfiles.d/systemd.conf\""
```

Comment:
According to the benchmark we also need to check ` /etc/tmpfiles.d/systemd.conf` if it exists.
So I'd do:
check-alternative #1:  /etc/tmpfiles.d/systemd.conf is 0640
check-alternative #2:  /usr/lib/tmpfiles.d/systemd.conf is 0640 and /etc/tmpfiles.d/systemd.conf doesn't exist
check-alternative #3:  neither file exists